src/ImageManipulation.php

```diff
@@ -516,7 +516,7 @@ public function PreviewThumbnail()
      */
     public function Thumbnail($width, $height)
     {
-        return $this->Pad($width, $height);
+        return $this->Fill($width, $height);
```

Comment:
I'm not sure about using `Fill()`, you lose the aspect ratio of the image uploaded...
Maybe `Fit()` would be better for this? Resizes to within the given width and height, but keeps aspect ratio.